tests/e2e/tests/checkout/card-failures.spec.js

```diff
@@ -0,0 +1,47 @@
+const { test, expect } = require( '@playwright/test' );
```

Comment:
It looks like we're not using `require` throughout the repo - except for webpack.config and the release script.

Can we use only `import` over `require`, for consistency, or is there a reason to use `require` in some places?

Reply:
It's just that I started re-using the code from [#woocommerce/](https://github.com/woocommerce/woocommerce) and they were using `require` on their codebase. I've replaced all `require` with `import`.